src/_tests/conftest.py

```diff
@@ -32,7 +39,80 @@ def client(context):
     "Fixture for tests which only read data"
     client = from_context(context)
     recons_container = client.create_container("reconstructions")
-    recons_container.write_array(np.zeros((2, 3, 3), dtype=np.int8), key="recon1")
-    masks_container = client.create_container("uid0001", metadata={"mask_idx": ["1"]})
-    masks_container.write_array(np.ones((1, 3, 3), dtype=np.int8), key="mask")
+    recons_container.write_array(
+        np.random.randint(0, 256, size=(5, 6, 6), dtype=np.uint8), key="recon1"
+    )
+    masks_container = client.create_container(
+        "uid0001", metadata={"mask_idx": ["1", "3"]}
+    )  # 2 slices
+    masks_container.write_array(np.ones((2, 6, 6), dtype=np.int8), key="mask")
     yield client
+
+
+@pytest.fixture
+def tiled_dataset(client):
+    tiled_dataset = TiledDataset(
+        data_tiled_client=client["reconstructions"]["recon1"],
+        mask_tiled_client=client["uid0001"],
+        is_training=True,
+        is_full_inference=False,
+    )
+    yield tiled_dataset
+
+
+@pytest.fixture(params=[
+    "src/_tests/example_tunet.yaml",
+    # "src/_tests/example_msdnet.yaml",
+    # "src/_tests/example_tunet3plus.yaml",
+    # "src/_tests/example_smsnet_ensemble.yaml",
+    # pytest.param("src/_tests/example_bad_params.yaml", marks=pytest.mark.bad_params)
+])
+def parameters_dict(request):
+    yaml_path = request.param
+    with open(yaml_path, "r") as file:
+        parameters_dict = yaml.safe_load(file)
+    yield parameters_dict
+
+@pytest.fixture
+def io_parameters(parameters_dict):
```

Comment:
I think we only want one fixture called `parameters_dict`, and remove `io_parameters`, `network_name`, `model_parameters`.

Reply:
In my first iteration I only have one fixture for all 3 sets of the parameters, then as the development evolving I started to see different scenarios where only one or two sets are used at once, and I made this work-around to reduce the redundancy in other parts of the code. But I am happy to change back to a single fixture.

Reply:
But you can get it easily with just the `parameters_dict` fixture, no?